### PR TITLE
Add 'x-checker-data' for Epiphany

### PR DESCRIPTION
**Pull request by philipstewart**
Flathub's external data checker should automatically check for updates and make a pull request

### CODE REVIEW

**Comment by philipstewart** [comment]
As per discourse, happy to extend to dependencies, if desired

**Comment by mcatanzaro** [comment]
LGTM, I'll let @bilelmoussaoui handle the final review though.

File: org.gnome.Epiphany.json [hunk 1]
```diff
@@ -82,7 +90,11 @@
                                 {
                                     "type": "git",
                                     "url": "https://github.com/sass/libsass.git",
```
Comment (bilelmoussaoui) [inline org.gnome.Epiphany.json:92]: I don't think libsass/sassc would change anytime in the future, but it is fine to keep them i guess

Comment (philipstewart) [reply org.gnome.Epiphany.json:92]: Agree, included for completeness

**Comment by bilelmoussaoui** [comment]
Let us see how this goes, thanks!

**Pull request merged**